src/Versity.Users/External/Presentation/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+﻿version: "3"
+
+services: 
```

Comment:
Do you have your service in docker?

Reply:
Yes, but not in this commit. But in this [amazing commit](https://github.com/jejikeh/versity/pull/3/commits/e9ea0e496fd4dd2936c4d3905b17f40e8a392916) it is.